pkg/splunk/common/util.go

```diff
@@ -424,6 +424,21 @@ func sortAndCompareSlices(a interface{}, b interface{}, keyName string) bool {
 		return true
 	}
 
+	// Sort slices
+	SortSlice(a, keyName)
+	SortSlice(b, keyName)
+
+	return !reflect.DeepEqual(a, b)
+}
+
+// SortSlice sorts a slice of any kind by keyName
+func SortSlice(a interface{}, keyName string) {
```

Comment:
Seems like needs a return type?

Reply:
I didn't find a need to have a return type here. Any errors experienced here leads to a panic.

Reply:
My bad, I didn't notice that return @ 463 is part of the anonymous function. Mya be better readable :-) ?